modules/src/ics23_commitment/mod.rs

```diff
@@ -17,11 +18,11 @@ impl CommitmentPath {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-pub struct CommitmentProof;
-
+pub type CommitmentProof = abci::Proof;
+/*
 impl CommitmentProof {
     pub fn from_bytes(_bytes: &[u8]) -> Self {
         todo!()
     }
 }
+*/
```

Comment:
Shouldn't we delete this but but extend the `abci::Proof` type instead?

Reply:
Yes, but will do in a separate issue/PR